docs/modules/ROOT/pages/how-tos/local-env.adoc

```diff
@@ -0,0 +1,198 @@
+= Running the Lieutenant Operator locally
+
+Running the Lieutenant Operator locally can be very helpful while debugging.
+This guide will run you through the necessary steps to do so.
+
+== Requirements
+
+Before you start, please make sure to have these requirements available:
+
+* A local Kubernetes cluster (`k3s`), managed with https://k3d.io/[k3d] or https://docs.docker.com/docker-for-mac/[Docker Desktop for Mac]
+** Alternatively you could use any other Kubernetes cluster, as long as you have administrator access.
+* A https://gitlab.com[GitLab.com] account 
+** Alternatively you could use your own GitLab instance (some adjustments need to be made in the guide)
+* Clone the Lieutenant Operator repository `https://github.com/projectsyn/lieutenant-operator`
+** If not specified otherwise all steps are performed at the root of the repository
+* A recent https://golang.org/doc/install[Go version] 
+* The following commands must work on your shell: `make`, `helm`, `kubectl`, `base64`
+
+[TIP]
+.Getting started on Linux with k3d
+--
+. Download `k3d` (version >= 3.x) from https://github.com/rancher/k3d/releases[GitHub]
+. Create a new cluster
++
+[source,shell]
+----
+k3d cluster create lieutenant
+----
+. Check that you're connected to this cluster
++
+[source,shell]
+----
+kubectl cluster-info
+----
+--
+
+[TIP]
+.Getting started on macOS with Docker Desktop for Mac
+--
+. Use https://docs.docker.com/docker-for-mac/[Docker Desktop for Mac] and enable https://docs.docker.com/docker-for-mac/#kubernetes[Kubernetes]
+--
+
+
+== Preparing the Cluster
+
+. Create a namespace to host the Operator
++
+[source,shell]
+----
+kubectl create namespace lieutenant
+----
+
+. Install the CRDs 
++
+[source,shell]
+----
+make install
+----
```

Comment:
we already have a Kubernetes setup with our e2e tests, invoked simply with `make e2e-test`. Why don't we re-use this instead of requiring more tools and having to invoke commands manually when it's already automated?
* It creates the cluster
* It prepares a kubeconfig in the local folder
* It installs the CRDs
* (It also install Lieutenant in `lieutenant-system` namespace, but I would be fine with separating e2e tests from a debug environment.)

Reply:
I see your point with unnecessary additional tools. 

I switched to a kind setup. I did not reuse everything from the e2e test, I think this is more usable as e debug environment, and I did not want to add a dependency on npm if not necessary..